frontend/play/index.js

```diff
@@ -784,7 +784,8 @@ function clamp(val, min, max) {
 }
 
 function initEditor() {
-  editor = new Editor(".editor", { sessionKey: "evy-editor", highlighter: highlightEvy })
+  const options = { sessionKey: "evy-editor", highlighter: highlightEvy, id: "evy-editor" }
```

Comment:
nit: this order of object fields is different to that of ai/nodex.js  - neither are alphabetical, and I'm not sure what the right order would be for an importance-based order.

Reply:
I'll make them alphabetical, but not sure what you mean with `ai/nodex.js`

Reply:
nvm. found it.